setup.py

```diff
@@ -28,24 +28,23 @@ def findfiles(pat):
             ],
         },
     install_requires=[
-        'path.py',
-        'argparse',
-        'pygeoip',
-        'pytz',
-        'python-dateutil',
-        'geoip2',
-        'lxml',
-        'BeautifulSoup',
-        'unicodecsv',
-        'Jinja2',
-        'google-api-python-client',
+        'path.py==11.5.2',
+        'pygeoip==0.3.2',
+        'pytz==2019.1',
+        'python-dateutil==2.8.0',
+        'geoip2==2.9.0',
+        'lxml==4.3.4',
+        'BeautifulSoup==3.2.1',
+        'unicodecsv==0.14.1',
+        'Jinja2==2.10.1',
+        'google-api-python-client==1.7.9',
         'edxcut',
```

Comment:
why was this excluded ?

Reply:
edX2bigquery requires installing edxcut by hand, since edxcut has a problem with its setup package. @andrey-canon